files/commands/seed_db.py

```diff
@@ -70,22 +68,22 @@ def hash_password(self, password):
 				"ban_evade":0,
 				"profileurl":"/e/feather.webp"
 			})
-			db.session.add(user)
+			db.add(user)
 		users_by_id[user_id] = user
 
-	db.session.commit()
-	db.session.flush()
+	db.commit()
+	db.flush()
```

Comment:
This is unnecessary: https://docs.sqlalchemy.org/en/20/orm/session_basics.html#committing
> The Session.commit() operation unconditionally issues Session.flush() before emitting COMMIT on relevant database connections. If no pending changes are detected, then no SQL is emitted to the database. This behavior is not configurable and is not affected by the Session.autoflush parameter.

Reply:
https://github.com/themotte/rDrama/issues/509 registered this one :)